### PR TITLE
more descriptive error message for non-existent packages

### PR DESCRIPTION
**Pull request by iamstarkov**
I'm using `latest-version` with `Promise.all` to request npm packages and some of them can be unavailable on npm, so current implementation show just that `package doesn't exist` but i requested bunch of packages, so it unclear which one doesn't exist. This small fix will clarify name of non-existing package.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -50,9 +50,8 @@ module.exports = function (name, version) {
 		})
 		.catch(function (err) {
 			if (err.statusCode === 404) {
-				throw new Error('Package doesn\'t exist');
+				throw new Error('Package "' + name + '" doesn\'t exist');
 			}
-
```
Comment (sindresorhus) [inline index.js]: Don't do unrelated changes.

File: index.js [hunk 2]
```diff
@@ -50,9 +50,8 @@ module.exports = function (name, version) {
 		})
 		.catch(function (err) {
 			if (err.statusCode === 404) {
-				throw new Error('Package doesn\'t exist');
+				throw new Error('Package "' + name + '" doesn\'t exist');
```
Comment (sindresorhus) [inline index.js]: Use 

```
`
```

instead of `"`

**Comment by sindresorhus** [comment]
Can you squash the commits?

File: index.js (hunk 1 above)
Comment (iamstarkov) [inline index.js]: sorry, it was by mistake

File: index.js (hunk 2 above)
Comment (iamstarkov) [inline index.js]: sure

**Comment by iamstarkov** [comment]
fixed and squashed

File: index.js (hunk 1 above)
Comment (sindresorhus) [inline index.js]: Just a tip. I always look through the diff before submitting a PR. That has caught a lot of mistakes in the past.
